MIGRATION.md

```diff
@@ -508,14 +596,12 @@ import * as RouterActions from './actions/router';
 @Injectable()
 export class RouterEffects {
   @Effect({ dispatch: false })
-  navigate$ = this.actions$
-    .ofType(RouterActions.GO)
-    .pipe(
-      map((action: RouterActions.Go) => action.payload),
-      tap(({ path, query: queryParams, extras }) =>
-        this.router.navigate(path, { queryParams, ...extras })
-      )
-    );
+  navigate$ = this.actions$.ofType(RouterActions.GO).pipe(
```

Comment:
this.actions$.ofType(RouterActions.GO).pipe(map...  >>>  this.actions$.pipe(ofType(RouterActions.GO), map(() => ...))

Reply:
I don't think this is necessary since these are the v4 docs.